demos/homeassistant DSO schedules/demo_instructions.md

```diff
@@ -0,0 +1,684 @@
+# Home Assistant with DSO profiles
+These demo instructions describe a proof-of-concept set-up to get capacity profiles communicated by a DSO (Distribution System Operator, such as Liander) available within Home Assistant. After following these instructions you should be able to reproduce the video below.
+
+https://github.com/alliander-opensource/ReLevENT/assets/58041016/8ab08547-4a0c-4b70-a9d6-2518d192b12d
+
+Please note these are *demo* instructions, as of February 2024 this solution is not in production.
+
+<!-- TOC -->
+  * [Assumed pre-installed configurations](#assumed-pre-installed-configurations)
+  * [Initial configuration](#initial-configuration)
+    * [Configuring a local MQTT broker (optional)](#configuring-a-local-mqtt-broker-optional)
+    * [Connecting to the MQTT broker](#connecting-to-the-mqtt-broker)
+      * [Direct connection](#direct-connection)
+      * [Bridged connection](#bridged-connection)
+    * [Setting up an MQTT sensor](#setting-up-an-mqtt-sensor)
+  * [Creating an automation](#creating-an-automation)
+  * [Visualization](#visualization)
+    * [History only using build-in card](#history-only-using-build-in-card)
+    * [History and future using apex-charts](#history-and-future-using-apex-charts)
+  * [Demo time!](#demo-time)
+<!-- TOC -->
+
+## Assumed pre-installed configurations
+Before we start, you are assumed to have the following working:
+1. A Hedera endpoint for which capacity profiles are published #todo link to Daniel
+2. A Hedera gateway to request capacity profiles (optional) #todo link to Daniel
```

Comment:
todo: add links from documentation from Daniel

Reply:
perhaps change phrasing such that Hedera is not absolutely necessary

Reply:
@fhg-dw @JakobVogelsang this is where I would like to link to some other documentation. Can you help me with this?
If we need to change the phrasing to be more generic, that is okay. Please do a suggestion in that case.